validation/validate.go

```diff
@@ -68,7 +68,11 @@ type SpecValidationError struct {
 }
 
 func (s StepValidationError) Error() string {
-	return fmt.Sprintf("%s:%d %s => '%s'%s", s.fileName, s.step.LineNo, s.message, s.step.GetLineText(), s.suggestion)
+	return fmt.Sprintf("%s:%d %s => '%s'", s.fileName, s.step.LineNo, s.message, s.step.GetLineText())
+}
+
+func (s StepValidationError) Suggestion() string {
+	return fmt.Sprintf("%s : %s => '%s'\t%s", "One or more errors were due to", "Step implementation not found", s.step.GetLineText(), s.suggestion)
```

Comment:
Why the suggestion message `Step implementation not found` is hardcoded? The error type is `StepValidationError` that means it can be used for both`step implementation not found` and `duplicate step implementations found` errors.